.gitignore

```diff
@@ -5,6 +5,11 @@
 npm
 npm-debug.log
 node_modules/
+package-lock.json
```

Comment:
package-lock.json is a file that we should commit. I would suggest removing this line.

Reply:
https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5